policyengine_uk/variables/gov/dwp/targeted_childcare_entitlement/targeted_childcare_entitlement_combine_benefits.py

```diff
@@ -0,0 +1,51 @@
+from policyengine_uk.model_api import *
+
+
+class targeted_childcare_entitlement_combine_benefits(Variable):
```

Comment:
A rule of thumb to exercise here is 'if I take out underscores and replace with spaces', this should fit natural into "I'm going to calculate this household's {variable name}".

This one doesn't